src/content/13/en/part13a.md

```diff
@@ -505,7 +505,7 @@ Save the SQL-commands you used at the root of the application repository in the
 
 #### Exercise 13.3.
 
-Create functionality in your application, which prints the blogs in the database using the command-line, e.g. as follows:
```

Comment:
Using this phrasing conveys something more along the lines of a command line app. (being able to handle command line inputs...)
This confused me, as I'm sure that was not intended.

Reply:
okay